Packs/CommonTypes/IncidentFields/incidentfield-cloudresourcelist.json

```diff
@@ -0,0 +1,31 @@
+{
+    "id": "incident_cloudresourcelist",
+    "version": -1,
+    "modified": "2022-09-28T13:34:47.376254979Z",
+    "name": "Cloud Resource List",
+    "ownerOnly": false,
+    "cliName": "cloudresourcelist",
+    "type": "multiSelect",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 0,
+    "hidden": false,
+    "openEnded": true,
+    "associatedTypes": [
```

Comment:
need to add a new type you created as well

Reply:
fixed.